walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -147,6 +148,34 @@ public void overlappingEvents() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void twoOverlaps() {
```

Comment:
It is recommended to make a test in the following structure:
preparation

function call

assert

In this case:
Collection<Event> events = Arrays.asList(...);
MeetingRequest request = ...;

Collection<TimeRange> actual = query.query(events, request);

Collection<TimeRange> expected =...
Assert.assertEquals(expected, actual);

Reply:
Are you proposing a new test, or proposing I should restructure this test? I think this particular test follows the structure you mention here.